src/OWML.Logging/OutputWriter.cs

```diff
@@ -15,5 +15,8 @@ public class OutputWriter : IModConsole
 
 		public void WriteLine(string line, MessageType type) => 
 			ConsoleUtils.WriteByType(type, line);
+
+		public void WriteLine(string line, MessageType type, string senderType) => 
+			ConsoleUtils.WriteByType(type, $"{senderType}: {line}");
```

Comment:
this should be fixed as `{modName}.{senderType}` probably

Reply:
idk

Reply:
actually this is only used by OWML in the launcher console , which means it's not used ;)
had to implement it as I added the method to the interface so mods (QSB) can access it easily.